db/fixtures/miq_product_features.yml

```diff
@@ -13,7 +13,8 @@
   :children:
   - :name: Accordions
     :description: All Accordions under Workloads
-    :feature_type: node
+    :feature_type: nodecd ../..
```

Comment:
@gtanzillo yes that's a typo i introduced during rebasing y branch, i will fix it, as i need to rebase this branch due to conflicts anyway.